core/src/test/java/org/elasticsearch/index/IndexWithShadowReplicasIT.java

```diff
@@ -86,6 +87,7 @@
 /**
  * Tests for indices that use shadow replicas and a shared filesystem
  */
+@LuceneTestCase.AwaitsFix(bugUrl = "fix this fails intermittently")
```

Comment:
I actually think these aren't needed any more, I think Boaz fixed the issue with these, they were removed from the `master` branch